modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidationFormBase.tsx

```diff
@@ -38,6 +39,20 @@ export default function ObjectValidationFormBase({
 				value="Groovy"
 			/>
 
+			<ClayForm.Group>
```

Comment:
@cgmonte @MariMalta Are you sure the description field should be displayed? Also, it seems that it's not possible to save a validation when you fill the description field with any value.